src/pages/Submission/Edit.js

```diff
@@ -11,8 +25,17 @@ export default () => {
     return unsubscribe
   }, [setIsSubmissionsOpen])
 
+  const submit = project => {
+    alert('TODO: handle form submit')
+    console.log(project)
+  }
+
   return (
     // TODO: Implement this page
-    isSubmissionsOpen ? <h1>TODO: Edit submission form</h1> : <h1>Edit - You cannot submit yet</h1>
+    isSubmissionsOpen ? (
+      <Form project={project} submitCallback={submit} onChange={setProject} />
+    ) : (
+      <h1>Edit - You cannot submit yet</h1>
```

Comment:
idk who's in charge of copy but maybe "Submissions are not open yet. Please check back later." H2?